tests/regression/lp-1808821/task.yaml

```diff
@@ -0,0 +1,16 @@
+summary: refreshing a snap with particular layout over itself
+details: |
+    This bug was hiding two separate bugs in snap-update-ns. The first
+    one is that Keep changes were not tracked in filesystem
+    assumptions, therefore did not participate in trespassing
+    detection correctly. A mount namespace that chose to reuse a tmpfs
+    (a keep mount change) would subsequently trigger a trespassing
+    violation for a path that was directly under the mounted tmpfs.
+    The second bug was that path creation code, which used a helper to
+    create prefix, passed a directory name of the prefix with the
+    trailing slash, which would not match the restrictive trespassing
+    detection code.
+execute: ./task.sh
```

Comment:
a bit unclear why is done this way?

Reply:
Mainly because it lets me run things in debug shell conveniently. There is some side improvement from getting IDE to understand the syntax of the test.